gpii/node_modules/testing/src/Mocks.js

```diff
@@ -366,7 +366,7 @@ gpii.test.integration.mockSettingsHandlerRegistry.populateOne = function (that,
 
 gpii.test.integration.mockSettingsHandlerRegistry.depopulate = function (that) {
     fluid.each(that.settingsHandlers, function (settingsHandler) {
-        if (settingsHandler.destroy) {
+        if (fluid.isComponent(settingsHandler)) {
```

Comment:
Is the other function still there?  You should be able to use this instead now.

Reply:
I'm not sure which one you mean by the "other function" - none of the other components host settingsHandlers so this implementation will not be relevant